### ISSUE TITLE
Suggestions for more data

### ISSUE THREAD

**Issue by Shiph** [petlof/liveresults#34]
- It would be nice to have a `country` field for each competition in `getcompetitions` to be able to sort competitions by country.
- It would also be very practical to have a function `getclubs` to get all clubs that compete in a competition (so you can then use `getclubresults`).

**Comment by callebokedal** [comment]
Yes, support for country codes would be very good. This definitions could be used, or suggested for admins:
https://www.loc.gov/standards/iso639-2/php/English_list.php

If introduced, perhaps `getcompetitions` could get a filtering option for this field as well. Nevertheless, adding this data field would enable the consumer of the API to filter by country code and that would be a great improvement!

Thanks in advance!
